src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/mixin/Credentials.java

```diff
@@ -33,7 +33,8 @@
      */
     public Credentials(String publickey, List<Entity> entities) {
         super(Identifiers.CREDENTIALS_SCHEME, Identifiers.CREDENTIALS, Identifiers.CREDENTIALS,
-                createAttributesSet(), new ArrayList<Action>(), new ArrayList<Mixin>(), setApplies(), entities);
+                createAttributesSet(), new ArrayList<Action>(), new ArrayList<Mixin>(), setApplies(),
```

Comment:
Explicit type arguments for ArrayList instantiation not required. They can be removed.